interface/main/messages/messages.php

```diff
@@ -334,6 +334,7 @@
 
 if ($noteid) {
     $body = preg_replace('/(:\d{2}\s\()'.$result['pid'].'(\sto\s)/','${1}'.$patientname.'${2}',$body);
+    $body = preg_replace('/\sto\s/i',' '.xl('to').' ',$body);
```

Comment:
Won't this translate every ' to ' within the note?
Would make much more sense (and would then be specific) to do the replace of to in the line above.